.github/workflows/cd-syftcli.yml

```diff
@@ -0,0 +1,102 @@
+name: CD - syft-cli
+
+on:
+  schedule:
+    - cron: "00 10 * * */3" # At 10:00 UTC on every three days
+
+  workflow_dispatch:
+    inputs:
+      skip_tests:
+        description: "If true, skip tests"
+        required: false
+        default: "false"
+
+jobs:
+  call-pr-tests-linting:
+    if: github.repository == 'OpenMined/PySyft' && (github.event.inputs.skip_tests == 'false' || github.event_name == 'schedule') # don't run on forks
+    uses: OpenMined/PySyft/.github/workflows/pr-tests-linting.yml@dev
+
+  call-pr-tests-syft:
```

Comment:
MAYBE: we could remove call-pr-tests-syft and call-pr-tests-stack, as currently, syft-cli is not coupled with syft and we could bring it back , once we have some base code?

Reply:
Sure! Good idea